### ISSUE TITLE
baseURL incorrect

### ISSUE THREAD

**Issue by i-tengfei** [kean/Get#45]
```swift
let baseURL = URL(string: "https://gitlab.com/api/v4")
URL(string: "/test", relativeTo: baseURL)?.absoluteString
```
expect
`https://gitlab.com/api/v4/test`

result
`https://gitlab.com/test`

<img width="630" alt="2022-07-14 05 19 14" src="https://user-images.githubusercontent.com/1763168/178838232-a4318ced-d551-484f-b1d6-152bc2e4f86b.png">

https://github.com/kean/Get/commit/73476a85cb69345886ac18e6d07c115bc6c9b00b

**Comment by kean** [comment]
Thanks for the report, @i-tengfei. I just switched to this API in the pre-release apparently fully without releasing how it worked. I'm going to revert the change. The `baseURL` should allow you to provide URLs with a path.

**Comment by kean** [comment]
Fix in [#46](https://github.com/kean/Get/pull/46). I added your example as a unit test.
